src/util/Constants.js

```diff
@@ -58,13 +58,15 @@ exports.DefaultOptions = {
   /**
    * WebSocket options (these are left as snake_case to match the API)
    * @typedef {Object} WebsocketOptions
-   * @property {number} [large_threshold=250] Number of members in a guild to be considered large
+   * @property {number} [large_threshold=50] Number of members in a guild to be considered large
    * @property {boolean} [compress=false] Whether to compress data sent on the connection
+   * @property {boolean} [guild_subscriptions=true] enables dispatching of guild subscription events (presence and typing events)
    * (defaults to `false` for browsers)
    */
   ws: {
-    large_threshold: 250,
+    large_threshold: 50,
     compress: false,
+    guild_subscriptions: true,
```

Comment:
This defaults to true, but this change would bring FAR too many (possibly breaking) changes to discord.js due to all the events that get stopped.

Reply:
true is the default, false disables it. normally all bots have this set to true.

Reply:
yes. what are you suggesting i change exactly?

Reply:
I think vlad thought true meant disable subscriptions

Reply:
That was a bad wording on my part, and I'm sorry. I know it defaults to true, and I meant that turning the flag off would bring far too many possibly breaking changes! 😅

Reply:
Isn't the ability to turn it off the whole point of this PR?